### PR TITLE
[iOS] Add 1.66.x translations.

### PR DESCRIPTION
**Pull request by iccub**
Resolves https://github.com/brave/brave-browser/issues/38342

## Submitter Checklist:

- [ ] I confirm that no [security/privacy review is needed](https://github.com/brave/brave-browser/wiki/Security-reviews) and no other type of reviews are needed, or that I have [requested](https://github.com/brave/reviews/issues/new/choose) them
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:

Verify that the Autofill Provider UI is translated

### CODE REVIEW

**Review by Brandon-T — APPROVED** [review]

File: ios/brave-ios/Sources/CredentialProviderUI/Resources/zh.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+"CredentialProvider.authenticationUnlockButtonTitle" = "解锁";
+
+/* A button that cancels the autofill request and dismisses the screen */
+"CredentialProvider.cancelButtonTitle" = "取消";
+
+/* A title for a button that when tapped copies the content highlighted into the users clipboard */
+"CredentialProvider.copyMenuItemTitle" = "复制";
+
+/* The title shown at the top of a list of credentials. 'Brave' in this case is the company name */
+"CredentialProvider.credentialListTitle" = "Brave 密碼";
```
Comment (nuo-xu) [inline ios/brave-ios/Sources/CredentialProviderUI/Resources/zh.lproj/Localizable.strings:14]:
```suggestion
"CredentialProvider.credentialListTitle" = "Brave 密码";
```

File: ios/brave-ios/Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings [hunk 2]
```diff
@@ -232,12 +235,18 @@
 /* The text beside the striked-through percentage Brave would normally charge for a swap. */
 "wallet.braveSwapFree" = "免费";
 
+/* A description of an Bitcoin account, displayed in Accounts tab. */
+"wallet.btcAccountDescription" = "比特幣";
```
Comment (nuo-xu) [inline ios/brave-ios/Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings:239]:
```suggestion
"wallet.btcAccountDescription" = "比特币";
```

File: ios/brave-ios/Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings [hunk 3]
```diff
@@ -265,6 +274,12 @@
 /* The title of the rank of this coin from market. */
 "wallet.coinMarketRank" = "排名";
 
+/* One of the coin types for users to create an account to store BTC assets */
+"wallet.coinTypeBitcoin" = "比特幣";
```
Comment (nuo-xu) [inline ios/brave-ios/Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings:278]:
```suggestion
"wallet.coinTypeBitcoin" = "比特币";
```

File: ios/brave-ios/Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings [hunk 4]
```diff
@@ -1267,6 +1294,9 @@
 /* The title on the restore wallet screen. */
 "wallet.restoreWalletTitle" = "汇入一个现有的钱包";
 
+/* The title for the button to retry a failed transaction. */
+"wallet.retryTransactionButtonTitle" = "重試交易";
```
Comment (nuo-xu) [inline ios/brave-ios/Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings:1298]:
```suggestion
"wallet.retryTransactionButtonTitle" = "重试交易";
```

**Review by nuo-xu — CHANGES_REQUESTED** [review]

File: ios/brave-ios/Sources/BraveWallet/Resources/tr.lproj/BraveWallet.strings [hunk 5]
```diff
@@ -559,6 +577,12 @@
 /* The button title for user to click to copy the account address for deposit. */
 "wallet.depositAddressCopy" = "Kopyala";
 
+/* The description of a swap button on the buy/send/swap/deposit modal */
+"wallet.depositDescription" = "Varlıkları yatırın";
+
+/* The disclosure at the bottom of the deposit view for ETH address indicates ETH wallet supports Ethereum Mainnet as well as the level 2 EVM. `%@` will be replaced with current supported level 2 EVM network names. */
+"wallet.depositEthDisclosure" = "Bu, Ethereum Ağını ve %@ gibi 2. seviye ağlarını destekleyen Ethereum cüzdanıdır";
```
Comment (soner-yuksel) [inline ios/brave-ios/Sources/BraveWallet/Resources/tr.lproj/BraveWallet.strings:584]: "Bu, Ethereum Ağını ve %@ gibi 2. seviye ağları destekleyen Ethereum cüzdanıdır";

File: ios/brave-ios/Sources/BraveWallet/Resources/tr.lproj/BraveWallet.strings [hunk 6]
```diff
@@ -1711,12 +1741,18 @@
...
+"wallet.solanaWithdrawNonceAccountInstructionName" = "Tek Seferlik Hesaptan Para Çekin";
+
 /* The description label of the sort option that sorts assets by their fiat value, shown below the title. Used in Portfolio/NFT filters and display settings. */
 "wallet.sortAssetsDescription" = "İtibari değer veya ada göre sırala";
 
 /* The label of the sort option that sorts assets by their fiat value. Used in Portfolio/NFT filters and display settings. */
 "wallet.sortAssetsTitle" = "Varlıkları Sırala";
 
+/* The title for the button to speed up a submitted transaction. */
+"wallet.speedUpTransactionButtonTitle" = "Speed Up";
```
Comment (soner-yuksel) [inline ios/brave-ios/Sources/BraveWallet/Resources/tr.lproj/BraveWallet.strings:1754]: "wallet.speedUpTransactionButtonTitle" = "Hızlandır";

**Review by soner-yuksel — APPROVED** [review]

**Review by kjozwiak — APPROVED** [review]
Uplift into `1.66.x` approved 👍

**Pull request merged**